### DISCUSSION TITLE
why, Why, WHY, WHY!!!

### DISCUSSION THREAD

**Discussion by SampleProvider** [spsquared/BattleBoxes_Multiplayer#110] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)

**Comment by spsquared** [comment, marked as answer]
WHY NOT
